Makefile

```diff
@@ -179,11 +179,11 @@ vet: ## Run go vet against code.
 
 .PHONY: ci-lint
 ci-lint: golangci-lint
-	$(GOLANGCI_LINT) run --timeout 15m0s
+	find . -path ./site -prune -o -name go.mod -execdir $(GOLANGCI_LINT) run $(GOLANGCI_LINT_FIX) --timeout 15m0s --config "$(PROJECT_DIR)/.golangci.yaml" \;
```

Comment:
why not just `find . -name go.mod`?

Reply:
I don’t think we need to check `./site`, so I believe this hack is acceptable.

Reply:
> I don’t think we need to check ./site, so I believe this hack is acceptable.

for sure, but site still appears in the output of 
```
> find . -path ./site -prune -o -name go.mod 
./site
./cmd/kueueviz/backend/go.mod
./go.mod
./hack/internal/tools/go.mod
```
it would be cleaner to exlude it completely to avoid confusion

Reply:
```
find . -path ./site -prune -o -name go.mod -execdir pwd \;
/Users/mykhailo_bobrovskyi/Projects/epam/kueue/cmd/kueueviz/backend
/Users/mykhailo_bobrovskyi/Projects/epam/kueue
/Users/mykhailo_bobrovskyi/Projects/epam/kueue/hack/internal/tools
```

Reply:
Added -false to do not print ./site.